### PR TITLE
volume health monitoring update

### PR DESCRIPTION
**Pull request by Sakshi-dell**
# Description
A few sentences describing the overall goals of the pull request's commits.
Documentation for volume health monitoring in CSI-PowerScale
# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|dock PR for https://github.com/dell/csm/issues/74 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/csidriver/features/powerscale.md [hunk 1]
```diff
@@ -527,3 +527,10 @@ In the case of operator-based installation, default permission for powerscale di
 
 Other ways of configuring powerscale volume permissions remain the same as helm-based installation.
 
+## PV/PVC Metrics
+
+CSI Driver for Dell EMC Powerscale 2.1.0 and above supports volume health monitoring.This allows Kubernetes to report on the condition of the underlying volumes via events when a volume condition is abnormal.
```
Comment (randeepsharma) [inline content/docs/csidriver/features/powerscale.md:532]: use PowerScale instead of Powerscale , S to be caps

**Review by randeepsharma — APPROVED** [review]

**Comment by bpjain2004** [comment]
@Sakshi-dell , we need to add few more documentation details about this feature once the PR https://github.com/dell/csi-powerscale/pull/32 is merged.

**Review by shanmydell — APPROVED** [review]
Changes look good to me. @Sakshi-dell: Please follow contribution guidelines for the repository henceforth.

File: content/docs/csidriver/features/powerscale.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/csidriver/features/powerscale.md]: Addressed

**Review by randeepsharma — DISMISSED** [review]

**Review by shanmydell — CHANGES_REQUESTED** [review]
Optional parameter changes can go as part of this PR after the PR in Repo is merged

**Review by randeepsharma — APPROVED** [review]
LGTM

**Review by randeepsharma — APPROVED** [review]
LGTM

**Review by prablr79 — APPROVED** [review]

**Pull request merged**